### ISSUE TITLE
My project for Covid19

### ISSUE THREAD

**Issue by Alessio-Siciliano** [jeremykohn/rid-covid#3]
Hello, I am developing a Convolutional Neural Network to diagnosis covid19 from recorded coughs. If you want, add my repository to your list 👍 
link: https://github.com/XxidroxX/Covid19-Diagnosis-from-recorded-coughts

Best regards, Alessio

**Comment by jeremykohn** [comment]
Hi Alessio! Looks like an interesting project. Glad you're working on it and hopefully it'll be useful in helping to combat the COVID-19 pandemic.

RID-COVID aims to detect/diagnose COVID-19 based on images including CT scans and X-rays. Audio recordings would be a different category -- maybe it's worthwhile to start a new list for audio-based diagnosis? I'm not sure how many other projects are using audio to diagnose COVID-19, but I quickly searched GitHub and found this one: https://github.com/zjyoung9490/Respiratory-Classification

**Comment by Alessio-Siciliano** [comment]
Yes I think that it is clearer to create a new category for deep learning audio-based diagnosis. The project you linked is very interesting, I will take a look at it as soon as I can.
